src/custom/components/TransactionSettings/TransactionSettingsMod.tsx

```diff
@@ -182,7 +183,11 @@ export default function TransactionSettings({ placeholderSlippage }: Transaction
         const parsed: number = Math.floor(Number.parseFloat(value) * 60)
         if (
           !Number.isInteger(parsed) || // Check deadline is a number
-          parsed < MINIMUM_ORDER_VALID_TO_TIME_SECONDS || // Check deadline is not too small
+          parsed <
```

Comment:
There's a lot of repetition in the code. Please, let keep the code DRYer


`tooLow` / `tooHigh` is hard to read, and has repetition.

Reply:
This code is not that different from original uni. I would rather not change it to keep it compatible.
The alternative is to break away completely but I don't want to that for this task